### ISSUE TITLE
Fails to retreive info in laters jellyfin 

### ISSUE THREAD

**Issue by mastercho** [LinFor/jellyfin-plugin-kinopoisk#6]
```
[00:39:55] [ERR] [77] MediaBrowser.Providers.TV.SeriesMetadataService: Error in КиноПоиск

System.Text.Json.JsonException: The JSON value could not be converted to System.Int32. Path: $.rating.ratingFilmCriticsVoteCount | LineNumber: 0 | BytePositionInLine: 5106.
---> System.InvalidOperationException: Cannot get the value of a token type 'Null' as a number.
at System.Text.Json.Utf8JsonReader.TryGetInt32(Int32& value)
at System.Text.Json.Utf8JsonReader.GetInt32()
at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
at System.Text.Json.JsonPropertyInfo`1.ReadJsonAndSetMember(Object obj, ReadStack& state, Utf8JsonReader& reader)
at System.Text.Json.Serialization.Converters.ObjectDefaultConverter`1.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
at System.Text.Json.JsonPropertyInfo`1.ReadJsonAndSetMember(Object obj, ReadStack& state, Utf8JsonReader& reader)
at System.Text.Json.Serialization.Converters.ObjectDefaultConverter`1.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
at System.Text.Json.Serialization.JsonConverter`1.ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
--- End of inner exception stack trace ---
at System.Text.Json.ThrowHelper.ReThrowWithPath(ReadStack& state, Utf8JsonReader& reader, Exception ex)
at System.Text.Json.Serialization.JsonConverter`1.ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
at System.Text.Json.JsonSerializer.ReadCore[TValue](JsonReaderState& readerState, Boolean isFinalBlock, ReadOnlySpan`1 buffer, JsonSerializerOptions options, ReadStack& state, JsonConverter converterBase)
at System.Text.Json.JsonSerializer.ReadAsync[TValue](Stream utf8Json, Type returnType, JsonSerializerOptions options, CancellationToken cancellationToken)
at Jellyfin.Plugin.Kinopoisk.KinopoiskApiProxy.GetAsync[T](Uri uri, CancellationToken cancellationToken)
at Jellyfin.Plugin.Kinopoisk.KinopoiskApiProxy.GetSingleFilm(Int32 filmId, Nullable`1 cancellationToken)
at Jellyfin.Plugin.Kinopoisk.KinopoiskVideoBaseProvider`2.GetMetadata(TLookupInfoType info, CancellationToken cancellationToken)
at MediaBrowser.Providers.Manager.MetadataService`2.ExecuteRemoteProviders(MetadataResult`1 temp, String logName, TIdType id, IEnumerable`1 providers, CancellationToken cancellationToken)
[00:39:55] [ERR] [77] MediaBrowser.Providers.TV.SeriesMetadataService: Error in КиноПоиск
```

I tried to search this https://www.kinopoisk.ru/series/842168/ by id and cant find it, Happens in lot of series. Fine in movies.

Jellyfin version 10.7.1
Unrad server : Linux

**Comment by LinFor** [comment]
Thanx, believe I fixed it in 10.7.0.2, try update

**Comment by mastercho** [comment]
Fixed Thanks!
